opentracing-spring-web/pom.xml

```diff
@@ -37,5 +42,17 @@
       <version>${version.org.springframework}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock-jre8</artifactId>
```

Comment:
I had to replace `MockRestServiceServer` in the tests with something else (WireMock), because `MockRestServiceServer` does not support `WebClient`.